doc/index.rst

```diff
@@ -145,16 +145,22 @@ If you are doing research using PennyLane, please cite
     *PennyLane: Automatic differentiation of hybrid quantum-classical computations.* 2018.
     `arXiv:1811.04968 <https://arxiv.org/abs/1811.04968>`_
 
-Support
--------
+Support and contribution
+------------------------
 
 - **Source Code:** https://github.com/XanaduAI/PennyLane
 - **Issue Tracker:** https://github.com/XanaduAI/PennyLane/issues
 
 If you are having issues, please let us know by posting the issue on our GitHub issue tracker.
 
-We also have a `PennyLane discussion forum <https://discuss.pennylane.ai>`_—come join the
-discussion and chat with our PennyLane team.
+We encourage contributions — simply fork the PennyLane repository, and then make a
+`pull request <https://help.github.com/articles/about-pull-requests/>`_ containing
+your contribution. All contributers to PennyLane will be listed as authors on the releases.
```

Comment:
Just a quick question, how come this was moved from ``research.rst`` (which was removed)?